src/modules/esl-toggleable/core/esl-toggleable.shape.ts

```diff
@@ -6,11 +6,19 @@ import type {ESLToggleable} from './esl-toggleable';
  * Used for TSX declaration
  */
 export interface ESLToggleableTagShape<T extends ESLToggleable = ESLToggleable> extends ESLBaseElementShape<T> {
-  /** Define CSS class to add on the body element */
+  /**
+   * Define CSS class to add on the body element
+   * @deprecated use 'related-active-class' instead
```

Comment:
```suggestion
   * @deprecated use 'container-active-class' instead
```

Reply:
BTW according to the latest discussion we are not going to deprecate body class